zeppelin-server/src/main/java/org/apache/zeppelin/socket/ConnectionManager.java

```diff
@@ -359,6 +359,19 @@ public void unicastParagraph(Note note, Paragraph p, String user, String msgId)
     }
   }
 
+  public static abstract class UserIterator {
+    public abstract void handleUser(String user, Set<String> userAndRoles);
+  }
```

Comment:
This class doesn't iterate a collection. Wouldn't the name `UserHandler` or `UserRoleHandler` be more appropriate?